### PR TITLE
update(excel) from 2016 to 2024 with new formulae

### PR DESCRIPTION
**Pull request by dannywinrow**
Remove:

FALSE|0
MIDBs
SQL.REQUEST
TRUE|0

Add:

ARRAYTOTEXT
BYCOL
BYROW
CHOOSECOLS
CHOOSEROWS
DROP
EXPAND
FALSE
FILTER
HSTACK
IMAGE
ISOMITTED
LAMBDA
LET
MAKEARRAY
MAP
MIDB
RANDARRAY
REDUCE
SCAN
SEQUENCE
SORT
SORTBY
STOCKHISTORY
TAKE
TEXTAFTER
TEXTBEFORE
TEXTSPLIT
TOCOL
TOROW
TRUE
UNIQUE
VALUETOTEXT
VSTACK
WRAPCOLS
WRAPROWS
XLOOKUP
XMATCH

### Checklist
- [ ] Added markup tests, or they don't apply here because...
- [ ] Updated the changelog at `CHANGES.md`

### CODE REVIEW

File: src/languages/excel.js [hunk 1]
```diff
@@ -263,19 +271,18 @@ export default function(hljs) {
...
     "ISPMT",
     "JIS",
     "KURT",
+    "LAMBDA",
     "LARGE",
     "LCM",
-    "LEFT",
-    "LEFTB",
-    "LEN",
-    "LENB",
```
Comment (joshgoebel) [inline src/languages/excel.js:278]: Please double check, these are included in the very link you provided.  MS is well known for forever-level-compatibility... I think if we're going to remove keywords we may need some explicit links/docs showing that microsoft has removed/deprecated certain functions.

Comment (dannywinrow) [reply src/languages/excel.js:278]: Apologies, I copied the wrong list over, thanks for picking up.  I'll fix and also make some comments about those being removed, or rlse just leave them all in.

Comment (dannywinrow) [reply src/languages/excel.js]: Ok, I've done a further commit to this branch which I hope resolves this.

**Comment by joshgoebel** [comment]
Better! you'll need to add an entry to CHANGES also.

File: CHANGES.md [hunk 2]
```diff
@@ -10,6 +10,7 @@ Core Grammars:
 - fix(makefile) - allow strings inside `$()` expressions [aneesh98][]
 - enh(arcade) updated to ArcGIS Arcade version 1.29 [Kristian Ekenes][]
 - enh(css) add all properties listed on MDN (96 additions including `anchor-name`, `aspect-ratio`, `backdrop-filter`, `container`, `margin-trim`, `place-content`, `scroll-timeline`, ...) [BaliBalo][]
+- enh(excel) add built-in functions for Excel 365 release to 2024 [dannywinrow][]
```
Comment (joshgoebel) [inline CHANGES.md:13]: You also need to add your link around line ~75 with all the others.

File: CHANGES.md [hunk 3]
```diff
@@ -89,6 +90,8 @@ CONTRIBUTORS
 [Jake Leahy]: https://github.com/ire4ever1190
 [Laurel King]: https://github.com/laurelthorburn
 [Kristian Ekenes]: https://github.com/ekenes
+[Danny Winrow]:
+https://github.com/dannywinrow
```
Comment (joshgoebel) [inline CHANGES.md:94]:
```suggestion
[Danny Winrow]: https://github.com/dannywinrow
```

**Review by joshgoebel — APPROVED** [review]

**Pull request merged**